lib/ReactViews/Tools/DiffTool/DiffTool.tsx

```diff
@@ -706,7 +734,7 @@ const DiffAccordion: React.FC<DiffAccordionProps> = props => {
           <Spacing right={1} />
           {/* font-size is non standard with what we have so far in terria,
               lineheight as well to hit nonstandard paddings */}
-          <Text css={"font-size: 17px;line-height: 26px;"} textLight>
+          <Text css={"font-size: 17px;line-height: 30px;"} textLight semiBold>
```

Comment:
Yes there is! Had just adjusted numbers as it was already there, but have since updated now in https://github.com/TerriaJS/terriajs/pull/4526/commits/8e4afbba3dc37855489f6494f99b4e7092e268fa#diff-07be940d3cdc8d9cc59750126772d9f9R742